src/main/scala/com/tsunderebug/scolor/otf/types/OTFEncodingRecord.scala

```diff
@@ -42,19 +42,18 @@ object OTFEncodingRecord {
     */
   case class SegmentedCoverageEncodingFormat(smg: Seq[SequentialMapGroup]) extends EncodingFormat {
 
-    override def getGlyphEntries: Map[Codepoint, GlyphID] = smg.map {
-        case SequentialMapGroup(s, e, g) =>
-          (s.toLong to e.toLong, g.toLong to (g + e - s).toLong)
-      }.flatMap {
-        case (cr, gr) =>
-          cr.zipWithIndex.map {
-            case (c, i) =>
-              (c, gr.drop(i.toInt).head)
+    override def getGlyphEntries: Map[Codepoint, GlyphID] = {
+      val emptyGlyph = UInt(0)
+      smg.foldLeft(Map.empty[Codepoint, GlyphID]) {
+        case (accum, SequentialMapGroup(startCharCode, endCharCode, startGlyphID)) => {
+          val glyphRange = (startGlyphID.toLong to (startGlyphID + endCharCode - startCharCode).toLong)
+          accum ++ (startCharCode.toLong to endCharCode.toLong).zipWithIndex.map {
+            //"Regardless of the encoding scheme, character codes that do not correspond to any glyph in the font should be mapped to glyph index 0. The glyph at this location must be a special glyph representing a missing character, commonly known as .notdef."
```

Comment:
Remove this comment here and add it in the scaladoc

Reply:
On the getGlyphEntries method or on the SegmentedCoverageEncodingFormat class?

Reply:
On the method :)

Reply:
There we go, how's that? a777f25